scripts/pr_link_docs.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+"""
+Script to generate a link to documentation preview in PRs.
+
+This is expected to be run by the `reusable_pr_link_docs.yml` GitHub workflow.
```

Comment:
I think it is time to put ci-related scripts under `scripts/ci` to distinguish them from scripts meant for humans.

Other candidates:

* generate_pr_summary.py
* version_util.py
* verify_wheels.py
* generate_prerelease_pip_index.py
* check_large_files.sh
* check_large_files_allow_list.txt
* …

Reply:
(but perhaps in a follow-up PR)

Reply:
It's a small thing, but https://github.com/rerun-io/rerun/issues/2292 so we don't forget